components/automate-ui/src/app/page-components/resources/resources.component.scss

```diff
@@ -1,6 +1,11 @@
 @import "~styles/variables";
 @import "~styles/mixins";
 
+#resource-item-list {
+  height: 20vh;
```

Comment:
I'm not sure the requirement, but this limits us to only seeing 3 resources at any given time without having to scroll.  UX wise that doesn't seem ideal given that there could be a lot of resources in there and users would have to scroll deep.  Looping in @susanev for a look